cluster/images/hyperkube/static-pods/kube-proxy.json

```diff
@@ -0,0 +1,27 @@
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "metadata": {
+    "name": "k8s-proxy",
+    "namespace": "kube-system"
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-proxy",
+        "image": "gcr.io/google_containers/hyperkube-ARCH:VERSION",
+        "command": [
+                "/hyperkube",
+                "proxy",
+                "--master=http://127.0.0.1:8080",
```

Comment:
Hi Lucas, I brought this issue up with Mike last week. I think part of the problem is that the bash script only runs kube-proxy on the workers if running <1.3.0-alpha.5 . I'm currently running 1.3.0-beta.1, but once beta.3 comes out with this PR maybe it'll be fixed. Though I suspect the bash script will need to be adjusted to run the kube-proxy again, since it's not in a DaemonSet any more.